dist/screeps-tests.ts

```diff
@@ -515,3 +515,11 @@ interface CreepMemory {
 
     BOOSTS[creep.body[0].type];
 }
+
+{
+    const tombstone = room.find(FIND_TOMBSTONES)[0];
+
+    Tombstone.creep.my;
```

Comment:
This will fail once the fix to the tombstone declaration is applied, right now this doesn't call properties of the constant declared above.